demo/core_13_$.utils.code.js

```diff
@@ -136,3 +136,60 @@ $.utils.code.toSourceSafe = function(value) {
     throw e;
   }
 };
+
+$.utils.code.formatForEval = function(src) {
+  // Eval treats {} as an empty block (return value undefined).
+  // Eval treats {'a': 1} as a syntax error.
+  // Eval treats {a: 1} as block with a labeled statement (return value 1).
+  // Detect these cases and enclose in parenthesis.
+  // But don't mess with: {var x = 1; x + x;}
+  // This is consistent with the console on Chrome and Node.
+  try {
+    var ast = $.utils.acorn.parse(src, { ecmaVersion: 5 });
```

Comment:
This set of options is already simplified compared to what we actually pass to acorn, and will become even more so as we begin to implement ES6 features.

Reply:
Changed the default options.